ihatemoney/tests/tests.py

```diff
@@ -178,7 +178,10 @@ def test_invite(self):
         self.client.get("/exit")
         # Test that we got a valid token
         resp = self.client.get(url, follow_redirects=True)
-        self.assertIn('You probably want to <a href="/raclette/add"', resp.data.decode('utf-8'))
+        self.assertIn(
+            'You probably want to <a href="/raclette/members/add"',
+            resp.data.decode('utf-8')
+        )
```

Comment:
Can you add a test checking that if members exists, the URL is proposing to add a new bill?

Reply:
of course... added :)

Reply:
I think tests should be more separate from each other. Here we're in the function testing that the invitation is working by email, so we probably don't want to change this one, and create two new test functions in order to test the new behavior. What do you think?